frontend/admin/src/js/components/user/UpdateUser.tsx

```diff
@@ -155,7 +155,7 @@ export const UpdateUserForm: React.FunctionComponent<UpdateUserFormProps> = ({
               })}
               type="email"
               name="email"
-              value={initialUser.email}
+              value={initialUser.email ?? ""}
```

Comment:
What's the purpose behind this change as well?

Reply:
This is a little weird because this is just for display, it's not submitted in the form.  I'm surprised but Maybe<string> (since email is nullable) cannot be assigned as undefined and Input.value can't be null.  This could be `initialUser.email ?? undefined` but I'm not sure that's any clearer.

Reply:
Got it, thanks for the explanation!